src/Maestro/tests/FeedCleaner.Tests/FeedCleanerTests.cs

```diff
@@ -0,0 +1,322 @@
+                            {
+                                new AssetLocation()
+                                {
+                                    Type = LocationType.NugetFeed,
+                                    Location = $"https://pkgs.dev.azure.com/{feed.Account}/{projectSection}_packaging/{feedName}/nuget/v3/index.json"
+                                }
+                            }
+                        });
+                    }
+
+                }
+            }
+
+            var context = GetContext();
+            context.Assets.AddRange(assets);
+            context.SaveChanges();
+        }
+
+        private Dictionary<string, AzureDevOpsFeed> SetupFeeds(string account)
+        {
+            
+            AzureDevOpsProject someProject = new AzureDevOpsProject("0", "someProject");
+            var allFeeds = new Dictionary<string, AzureDevOpsFeed>();
+
+            // This is the reference release feed.
+            var releaseFeed = new AzureDevOpsFeed(account, "0", "releaseFeed", someProject)
+            {
+                Packages = new List<AzureDevOpsPackage>()
+                {
+                    new AzureDevOpsPackage("releasedPackage1", "nuget")
+                    {
+                        Versions = new AzureDevOpsPackageVersion[]
+                        {
+                            new AzureDevOpsPackageVersion("1.0", false),
+                            new AzureDevOpsPackageVersion("2.0", true),
+                        }
+                    },
+                    new AzureDevOpsPackage("releasedPackage2", "nuget")
+                    {
+                        Versions = new AzureDevOpsPackageVersion[]
+                        {
+                            new AzureDevOpsPackageVersion("1.0", false),
+                            new AzureDevOpsPackageVersion("2.0", false),
+                        }
+                    }
+                }
+            };
+            allFeeds.Add("releaseFeed", releaseFeed);
+
+            var managedFeedWithUnreleasedPackages = new AzureDevOpsFeed(account, "1", "darc-int-some-repo-12345678", null)
+            {
+                Packages = new List<AzureDevOpsPackage>()
+                {
+                    new AzureDevOpsPackage("unreleasedPackage1", "nuget")
+                    {
+                        Versions = new AzureDevOpsPackageVersion[]
+                        {
+                            new AzureDevOpsPackageVersion("1.0", false)
+                        }
+                    },
+                    new AzureDevOpsPackage("releasedPackage1", "nuget")
+                    {
+                        Versions = new AzureDevOpsPackageVersion[]
+                        {
+                            new AzureDevOpsPackageVersion("1.0", false),
+                        }
+                    }
+                }
+            };
+            allFeeds.Add("darc-int-some-repo-12345678", managedFeedWithUnreleasedPackages);
```

Comment:
Perhaps the key used in the `Add` here could be a property from `managedFeedWithUnreleasedPackages` so we don't need to duplicate the literal "darc-int-some-repo-12345678"